### DISCUSSION TITLE
How do I run terminal commands in iTerm2 instead of default Terminal?

### DISCUSSION THREAD

**Discussion by Stooovie** [pilotmoon/PopClip-Extensions#1151] [Q&A] [answered]
How do I set up iTerm2 (3.4.4) as default for PopClip? Any command opens in default Terminal. Big Sur 11.2.2.

**Comment by pilotmoon** [comment, marked as answer]
Hiya, in PopClip extension prefs you can click the pencil, then the cog next to Terminal. Then there's a dropdown to select the terminal app.
<img width="154" alt="CleanShot 2021-03-10 at 18 36 10@2x" src="https://user-images.githubusercontent.com/17520/110679560-808efd00-81cf-11eb-8131-eac972532f29.png">
